include/ibm/management_console_rest.hpp

```diff
@@ -1,9 +1,10 @@
 #pragma once
```

Comment:
Could you add some more detail to the commit msg about exactly what you are doing here? 
Where it is from, etc.